compiler/compiler-settings.h

```diff
@@ -91,7 +91,11 @@ class CompilerSettings : vk::not_copyable {
     colored
   };
 
-  KphpOption<std::vector<std::string>> main_files;
+  // right now we require that there is only 1 file in the input
+  // files list: a main file, but we may allow more than 1 input
+  // files in future (?); it also makes giving an error for
+  // more than 1 positional argument easier
+  KphpOption<std::vector<std::string>> input_files;
```

Comment:
I don't think that we need to keep a vector of strings here and take a back element every time.
It seems that leaving only one file here would be better.

Reply:
I haven't found a simple way to give an error for the case where user provides more than 1 positional argument.
Maybe @AlexK0 can give a hint here?
My best bet was to modify the args parser and warn if number of positional arguments is greater than 1.

Reply:
Done.